src/BlazorSignalR/Internal/BlazorHttpConnection.cs

```diff
@@ -320,7 +332,17 @@ private static Uri CreateConnectUrl(Uri url, string connectionId)
 
         private HttpClient CreateHttpClient()
         {
-            HttpMessageHandler handler = new WebAssemblyHttpMessageHandler();
+            HttpMessageHandler handler;
+
+            if (_isServerSide)
+            {
+                handler = new HttpClientHandler();
```

Comment:
Does this just cause the server to send the SignalR requests, instead of getting the browser to? As this would be unexpected behaviour.

Reply:
Yes. Good point. The problem here is that `WebAssemblyHttpMessageHandler` is not available when not running on blazor. There is no http proxy that sends the request to the client that actually does the http request. We can either build this ourselves (based on the blazor code) or the server is responsible for the http request.  

> As this would be unexpected behaviour.

Why this?

Reply:
Creating our own ```HttpMessageHandler``` is probably going to be the best way forward. And to then simplify the code, just not use ```WebAssemblyHttpMessageHandler``` at all. This will also then remove the need for the ```isServerSide``` check.

If you wanted to open a SignalR connection from the server side, someone can implement whatever logic they want and just call the standard SignalR client constructors. However, the expectation when using ```BlazorSignalR``` to configure the client, is that all requests are originating from the client. Now in most cases where the request comes from won't make much difference, however if you were trying to access a local server behind a firewall, then it would make a big difference.

Reply:
Thats probably the best way going foreward. I will look into this the next days, if I have some spare time.